### PR TITLE
[Utils] Installer: Formatting and Better source message

### PR DESCRIPTION
**Pull request by SAtacker**
- Ref: https://github.com/WasmEdge/WasmEdge/issues/2687#issue-1820113967

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: utils/install.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def compare(self, version2, separator=". |-", ignorecase=True):
     WASMEDGE_TENSORFLOW_PLUGIN,
     WASMEDGE_TENSORFLOW_LITE_PLUGIN,
     WASMEDGE_IMAGE_PLUGIN,
-    WASM_BPF
+    WASM_BPF,
```
Comment (hydai) [inline utils/install.py:345]: I would like to know if we can have a Python linter in the test-installer.yml workflow.

**Review by hydai — CHANGES_REQUESTED** [review]

File: utils/install.py (hunk 1 above)
Comment (SAtacker) [reply utils/install.py:345]: Done, added it

**Comment by SAtacker** [comment]
@alabulei1 PTAL, thanks

**Comment by SAtacker** [comment]
CC @hydai

**Review by hydai — APPROVED** [review]

**Pull request merged**